File: lib/LocalModules/DevOther/src/main/java/java/dev/engine/cache/DataItem.java
package java.dev.engine.cache;

import dev.engine.cache.ICacheEngine;

/**
 * detail: Cache ( Data、Params ) Item
 * @author Ttt
 */
public class DataItem
        extends ICacheEngine.EngineItem {

    public DataItem(
            String key,
            int type,
            long size,
            long saveTime,
            long validTime,
            boolean isPermanent,
            boolean isDue
    ) {
        super(key, type, size, saveTime, validTime, isPermanent, isDue);
    }
}

File: lib/DevEngine/src/main/java/dev/engine/cache/DataItem.kt
package dev.engine.cache

/**
 * detail: Cache ( Data、Params ) Item
 * @author Ttt
 */
open class DataItem(
    key: String?,
    type: Int,
    size: Long,
    saveTime: Long,
    validTime: Long,
    isPermanent: Boolean,
    isDue: Boolean
) : ICacheEngine.EngineItem(key, type, size, saveTime, validTime, isPermanent, isDue)